components/checkbox/checkbox.js

```diff
@@ -8,16 +8,24 @@ function Checkbox(props) {
   const {
     checked,
     children,
+    dataAttrs = {},
     disabled,
     name,
     onChange,
+    ...otherProps
```

Comment:
In other components, for example, Dropdown, Autocomplete, Input, we already have possibility to directly pass necessary props like `data-gtm-id` to component, which is quite convenient and laconically.